### PR TITLE
PacketList.conversations(): add labels, support ARP by default

### PR DESCRIPTION
**Pull request by p-l-**

### CODE REVIEW

File: scapy/plist.py [hunk 1]
```diff
@@ -299,10 +306,15 @@ def conversations(self, getsrcdst=None,**kargs):
             except:
                 #XXX warning()
```
Comment (guedou) [inline scapy/plist.py]: Shall we remove this comment ?

**Comment by guedou** [comment]
@p-l- sorry for the delay ! I have a small comment. Except that, I am good to merge.

File: scapy/plist.py (hunk 1 above)
Comment (p-l-) [inline scapy/plist.py]: The comment was here before, I left it but I'm not sure. Do you want me to remove it?

Comment (guedou) [inline scapy/plist.py]: I think that we can remove it. Replacing it with a warning message might be better.

Comment (p-l-) [inline scapy/plist.py]: I have updated the comment to explain why it might not be ideal to add a warning here. What do you think?

**Pull request merged**